ros/src/computing/perception/detection/packages/cv_tracker/nodes/obj_reproj/obj_reproj.cpp

```diff
@@ -192,6 +200,32 @@ static visualization_msgs::MarkerArray convert_marker_array(const cv_tracker::ob
   return ret;
 }
 
+#ifdef IS_FOUND_JSK_PLUGIN
+static jsk_recognition_msgs::BoundingBoxArray convertJskBoundingBoxArray(const cv_tracker::obj_label& src)
+{
+  jsk_recognition_msgs::BoundingBoxArray ret;
+
+  for (const auto& reproj_pos : src.reprojected_pos)
+    {
+      jsk_recognition_msgs::BoundingBox bounding_box;
+      /* Set frame ID */
+      bounding_box.header.frame_id = "map";
+
+      /* set pose of bounding box  */
+      bounding_box.pose.position = reproj_pos;
+
+      /* set scale of bounding box */
+      bounding_box.dimensions.x = (double)1.5;
```

Comment:
Cast to `double` is not necessary. Floating point literal without suffix is `double` type.

- http://en.cppreference.com/w/cpp/language/floating_literal
- http://docs.ros.org/hydro/api/geometry_msgs/html/msg/Vector3.html